package.json

```diff
@@ -53,7 +53,6 @@
     "babel-plugin-transform-define": "^2.0.0",
     "babel-preset-cozy-app": "^1.9.0",
     "cozy-authentication": "1.19.1",
-    "cozy-intent": "^1.7.0",
```

Comment:
test/build will be broken if `cozy-ui@77.9.1` doesn't have access to `cozy-intent`